monkey/infection_monkey/post_breach/pba.py

```diff
@@ -12,57 +15,69 @@ class PBA(object):
     """
     Post breach action object. Can be extended to support more than command execution on target machine.
     """
-    def __init__(self, name="unknown", linux_command="", windows_command=""):
+    def __init__(self, name="unknown", command=""):
         """
         :param name: Name of post breach action.
-        :param linux_command: Command that will be executed on linux machine
-        :param windows_command: Command that will be executed on windows machine
+        :param command: Command that will be executed on breached machine
         """
-        self.linux_command = linux_command
-        self.windows_command = windows_command
+        self.command = command
         self.name = name
 
-    def run(self, is_linux):
+    @staticmethod
+    def get_pba():
         """
-        Runs post breach action command
-        :param is_linux: boolean that indicates on which os monkey is running
+        Should be overridden by all child classes.
+        This method returns a PBA object based on worm's configuration.
+        :return: An array of PBA objects.
```

Comment:
Currently, your code assumes it's a single PBA object rather than an array, fix docs to avoid confusing people.

Reply:
Also, if I understand correctly, I should return None if I decide not to return a PBA object? For example, something Windows specific on Linux? 
Lets document that